From fbb0584443336d90ba7ae95a67e8d35c6c15a5ac Mon Sep 17 00:00:00 2001
From: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Date: Thu, 10 Feb 2022 15:41:55 +0200
Subject: [PATCH 37/43] s32-gen1: soc: Remove enable_i2c_clk()

Removed a clock enable function no longer used in the
s32-gen1 i2c driver.

Issue: ALB-8352
Upstream-Status: Pending 

Signed-off-by: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm/cpu/armv8/s32/s32-gen1/soc.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm/cpu/armv8/s32/s32-gen1/soc.c b/arch/arm/cpu/armv8/s32/s32-gen1/soc.c
index 6a05b54fcd..febc3797bd 100644
--- a/arch/arm/cpu/armv8/s32/s32-gen1/soc.c
+++ b/arch/arm/cpu/armv8/s32/s32-gen1/soc.c
@@ -68,14 +68,6 @@ int cpu_numcores(void)
 	return hweight32(cpu_mask());
 }
 
-int enable_i2c_clk(unsigned char enable, unsigned i2c_num)
-{
-	if (enable)
-		return s32gen1_enable_plat_clk(S32GEN1_CLK_XBAR_DIV3);
-
-	return 0;
-}
-
 __weak int dram_init(void)
 {
 	return 0;
-- 
2.17.1

